pkg/virt-controller/services/rendervolumes.go

```diff
@@ -26,23 +27,25 @@ import (
 type VolumeRendererOption func(renderer *VolumeRenderer) error
 
 type VolumeRenderer struct {
-	containerDiskDir      string
-	ephemeralDiskDir      string
-	virtShareDir          string
-	namespace             string
-	vmiVolumes            []v1.Volume
-	podVolumes            []k8sv1.Volume
-	podVolumeMounts       []k8sv1.VolumeMount
-	sharedFilesystemPaths []string
-	volumeDevices         []k8sv1.VolumeDevice
-}
-
-func NewVolumeRenderer(namespace string, ephemeralDisk string, containerDiskDir string, virtShareDir string, volumeOptions ...VolumeRendererOption) (*VolumeRenderer, error) {
+	containerDiskDir              string
+	ephemeralDiskDir              string
+	virtShareDir                  string
+	namespace                     string
+	imageVolumeFeatureGateEnabled bool
```

Comment:
Nit:
```suggestion
	useImageVolumes bool
```

Supernit:
Move the new bool to the top of the struct to match the other orders

Reply:
Done